ccp/6_preflight_checks/checks.yml

```diff
@@ -0,0 +1,47 @@
+---
+- hosts: localhost
+  gather_facts: no
+  vars:
+    ses_config_file_path: "~/suse-osh-deploy/ses_config.yml"
+  tasks:
+    - name: Ensuring ses config is available
+      stat:
+        path: "{{ ses_config_file_path }}"
+      register: _sesconfigfile
+
+    - name: Exit if file doesn't exist
+      fail:
+        msg: "ses config not found"
+      when:
+        - not _sesconfigfile.stat.exists | bool
+    # Uncomment this when ses_config is decided to be the source of truth for OSD pools
+    #
+    #- name: Loading the ses variables
+    #  include_vars: "{{ ses_config_file_path }}"
+
+    #- name: Ensuring data of the ses_config_file_path contains the right information
+    #  debug:
+    #    var: item
+    #  with_items:
+    #    - "{{ ses_cluster_configuration['conf_options']['ses_mon_host'] }}"
+    #    - "{{ ses_cluster_configuration['cinder']['rbd_store_pool'] }}"
+    #    - "{{ ses_cluster_configuration['nova']['rbd_store_pool'] }}"
```

Comment:
yes, ses config yaml is the truth :-). I think we are missing the pools for cinder, cinder backup, glance, libvirt and nova. and keyring(s).

Reply:
At the time of the writing, we were not sure about said truth.
I think it's fine as first step (as this works) and do other PRs to add this feature.